server.go

```diff
@@ -422,11 +423,13 @@ func (s *Server) runHandler(handler HandlerFunc, deliveries <-chan amqp.Delivery
 		go func(delivery amqp.Delivery) {
 			handler(ctx, &rw, delivery)
 
-			s.responses <- processedRequest{
-				replyTo:    delivery.ReplyTo,
```

Comment:
Where did this end up before your if statement? 🤔

Reply:
Sent to the amqp-server, which just threw it away.